salt/_modules/metalk8s_monitoring.py

```diff
@@ -164,42 +162,45 @@ def get_alerts(state=None, **kwargs):
 
 
 def _requests_alertmanager_api(route, method="GET", **kwargs):
-    endpoint = __salt__["metalk8s_kubernetes.get_service_endpoints"](
+    endpoints = __salt__["metalk8s_kubernetes.get_service_endpoints"](
```

Comment:
So from what I see you do not use the new `get_service_ips_and_ports` function

Reply:
Fixed.